.travis.yml

```diff
@@ -48,6 +48,7 @@ script:
   - php scripts/pre-commit.php -q -l
   - php scripts/pre-commit.php -q -s
   - php scripts/pre-commit.php -u --db --snmpsim --fail-fast
+  - test -z "$BROWSER_TEST" || php artisan config:clear
```

Comment:
This is because dusk rewrites the .env file and it won't be picked up if it's cached.